packages/react-table/src/components/Table/utils/decorators/treeRow.tsx

```diff
@@ -0,0 +1,60 @@
+import * as React from 'react';
+import { IExtra, IFormatterValueType, OnCheckChange, OnTreeRowCollapse } from '../../TableTypes';
+import { css } from '@patternfly/react-styles';
+import styles from '@patternfly/react-styles/css/components/Table/table';
+import stylesTreeView from '@patternfly/react-styles/css/components/Table/table-tree-view';
+import { Button, Checkbox } from '@patternfly/react-core';
+import AngleDownIcon from '@patternfly/react-icons/dist/js/icons/angle-down-icon';
+
+export const treeRow = (onCollapse: OnTreeRowCollapse, onCheckChange?: OnCheckChange) => (
+  value: IFormatterValueType,
+  { rowIndex, rowData }: IExtra
+) => {
+  const { isExpanded, level, setsize, toggleAriaLabel, checkAriaLabel, isChecked, checkboxId } = rowData.props;
+  const text = (
+    <div className={css(stylesTreeView.tableTreeViewContent)}>
+      <span className="pf-c-table__text">{value.title || value}</span>
+    </div>
+  );
+  const onChange = (isChecked: boolean, event: React.FormEvent<HTMLInputElement>) => {
+    onCheckChange(event, isChecked, rowIndex, value.title, rowData);
```

Comment:
above you check for `value.title || value`, the same here then?
Maybe set a new variable `const content = value.title || value` and use that everywhere